public/pages/CreateMonitor/components/AssociateMonitors/AssociateMonitors.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Fragment, useState, useEffect, useCallback } from 'react';
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import MonitorsList from './components/MonitorsList';
+import { FormikCodeEditor } from '../../../../components/FormControls';
+import * as _ from 'lodash';
+import { isInvalid, hasError, validateExtractionQuery } from '../../../../utils/validate';
+
+const AssociateMonitors = ({
```

Comment:
Since this is a tsx file, we should have types defined for Props and other objects used in this file

Reply:
Yeah, I did that by default, I changed it to js as all other files are js.